src/app/examples/button-example/button-example.component.html

```diff
@@ -3,4 +3,5 @@
 <button kirby-button expand="block">Kirby Block Button</button>
 <button kirby-button expand="block" shape="square">Kirby Square Block Button</button>
 <button kirby-button colortype="primary">Primary Button</button>
-<button kirby-button colortype="secondary">Secondary Button</button>
\ No newline at end of file
+<button kirby-button colortype="secondary">Secondary Button</button>
+<button kirby-button shape="round" disabled>Disabled Button</button>
```

Comment:
We don't use `shape` anymore
```suggestion
<button kirby-button disabled>Disabled Button</button>
```

Reply:
Removed